recipes/about-time/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "about-time" %}
+{% set version = "4.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/about-time-{{ version }}.tar.gz
+  sha256: 963b1f3739b0c9732eb205031762b76f1291d89b5d0c8220a8d5b154e32ce650
+  patches:
+    - setup-with-encoding.patch
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
```

Comment:
Upstream [lists](https://github.com/rsalmei/about-time/blob/d3fb29d29362612f0c8b4343e745dfbcd241207a/setup.py#L48) Python >=3.7 ?

Reply:
Yes, you are right. I rerun the `grayskull` command and get the correct `python_required` attribute. Weird and thx.